src/tests/dfio/SConscript

```diff
@@ -0,0 +1,33 @@
+"""Build daos fio engine"""
+import os
+import daos_build
+
+def scons():
+    """Execute build"""
+
+    Import('env', 'prereqs')
+
+    libs = ['daos_common', 'daos', 'dfs', 'gurt', 'uuid', 'dl']
+
+    denv = env.Clone()
+
+    if prereqs.has_source(env, 'fio'):
+        print "Building DAOS FIO Engine with src: %s" % env.subst("$FIO_SRC")
```

Comment:
(pylint-syntax-error) invalid syntax

Reply:
ah, need to use print("...."....) or this will fail with python3 build

Add from __future__ import print_function at top